contracts/Library/LibraryLockDataLayout.sol

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.6.9;
+
+contract LibraryLockDataLayout {
+  bool public initialized = false;
```

Comment:
here you add a variable that is not part of the storage, could mess things up when we update the contract

Reply:
no it's an initialization variable, I used the exact implementation of [UUPS](https://eips.ethereum.org/EIPS/eip-1822) for this contract